test_scripts/Polices/Related_HMI_API/ATF_OnAllowSDLFunctionality_allowed_true_without_device.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@vvvakulenko As usual this script is not in scope to check this requirement.
Is the idea to not consent device and to check that when: 
"PoliciesManager receives SDL.OnAllowSDLFunctionality with 'allowed=false' and without 'device' param from HMI"
the update of consented section will not be changed.

If Yes - correct using comments of previous script.
If No - remove the scripts.